docs/docs_for_team/README.md

```diff
@@ -78,3 +78,103 @@ From Jenkins, 'New item' -> 'Pipeline'. Then, the settings are as follows:
 * Pipeline Definition: Pipeline script from SCM Pipeline
 * SCM: Git
 * Pipeline Repository URL: [git-repo-url]
+
+## Decommissioning of the Jenkins
+
+There are 4 steps to decommission the Jenkins platform for one of your environments:
+
+* decommissioning the Jenkins infrastructure, via `terraform destroy`
+
+* decommissioning the DNS infrastructure, via `terraform destroy`
+
+* deleting the S3 buckets used for the Terraform state files
+
+* deleting the Github OAuth app
+
+### Before you start
+
+1. Move to the directory in which you cloned the repository originally.\
+If you don't have it anymore, clone the repository again and customise the `terraform.tfvars`
+as you did during the provisioning steps, for both the `terraform\dns` and `terraform\jenkins` directories.
+
+1. Make sure you still have this in `~/.aws/credentials`, otherwise add it again:
+   
+       ```
+       [re-build-systems]
+       aws_access_key_id = [your aws key here]
+       aws_secret_access_key = [your aws secret here]
+       ```
+
+1. Export these environment variables
+
+    ```
+    export JENKINS_ENV_NAME=[environment-name]
+    export JENKINS_TEAM_NAME=[team-name]
+    ```
+    
+1. Export your credentials
+
+    ```
+    export AWS_ACCESS_KEY_ID="[aws key]"
+    export AWS_SECRET_ACCESS_KEY="[aws secret]"
+    export AWS_DEFAULT_REGION="eu-west-1"
+    ```
+
+### Decommissioning the Jenkins infrastructure
+
+1. Run these commands from the `terraform/jenkins` directory: 
+
+    ```
+    terraform init \
```

Comment:
I don't think we need this "terraform init" code block

Reply:
@smford I have tested it, and you are right, we don't need `terraform init`. Going to remove it. Thanks!